app/controllers/dashboard/dashboard-activity-controller.js

```diff
@@ -180,9 +164,10 @@ module.exports = (req, res) => {
   }
 }
 
-function getTransactionDateRange (period, customRange) {
-  let fromDateTime
-  let toDateTime = moment().tz('Europe/London').format() // Today is the default
+function getTransactionDateRange (period) {
```

Comment:
I'm a little concerned about such a big change to this method without any unit tests validating behaviour, are there some that I've just missed?

Also it looks like this removes any option for custom dates, is that definitely not used by anything?

Reply:
> I'm a little concerned about such a big change to this method without any unit tests validating behaviour, are there some that I've just missed?
`dashboard-activity-controller.test` 
> Also it looks like this removes any option for custom dates, is that definitely not used by anything?
It was going to be a feature 2-3 years ago but it was halted. The decision was that if the user wants specific dates then they should use the transaction page. I had a discussion with @jonheslop about it.

Reply:
> I'm a little concerned about such a big change to this method without any unit tests validating behaviour, are there some that I've just missed?
> 
> Also it looks like this removes any option for custom dates, is that definitely not used by anything?
 
It's tested under `dashboard-activity-controller.test`

The customRange was going to be a feature about 2-3 years ago, but it was halted by the designers. The decision was that if the user wants specific dates then they should use the transaction page. I had a discussion with @jonheslop about it.